web/components/templates/dashboard/dashboardPage.tsx

```diff
@@ -207,11 +207,11 @@ const DashboardPage = (props: DashboardPageProps) => {
   });
 
   useEffect(() => {
-    if (filterMap) {
+    if (!isAnyLoading && filterMap) {
       setAdvancedFilters(getAdvancedFilters());
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [filterMap]);
+  }, [isAnyLoading]);
```

Comment:
nice catch, should we also add filtermap to the deps list?

Reply:
noo because if selectedFilter.length === 1 (only one filter is selected) and you try to delete this filter it's not gonna work. We need to render page only once when got filterMap. Same thing is in Requests page but I forgot to change Dashboard in prev pr